### PR TITLE
added admx references

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#6483

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
could you please have a look?
Thank you

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/windowsdefenderapplicationguard-csp.md [hunk 1]
```diff
@@ -44,6 +53,15 @@ Determines the type of content that can be copied from the host to Application G
 
 <a href="" id="clipboardsettings"></a>**Settings/ClipboardSettings**  
 This policy setting allows you to decide how the clipboard behaves while in Application Guard. Value type is integer. Supported operations are Add, Get, Replace, and Delete
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/windowsdefenderapplicationguard-csp.md:55]:
```suggestion
This policy setting allows you to decide how the clipboard behaves while in Application Guard. Value type is integer. Supported operations are Add, Get, Replace, and Delete.
```

**Comment by mypil** [comment]
@ManikaDhiman - PR has been copy edited and is ready for final review. This is based on issue #6483.

Thank you.

cc: @kenwith

**Comment by ManikaDhiman** [comment]
Hi @mypil, I have reached out to the product team to verify and confirm these changes. Thanks!

**Comment by yogeshasalkar** [comment]
Any updates?

**Comment by illfated** [comment]
@yogeshasalkar : I think we need to wait a few more days, because the product team is always busy and it can take more than just a couple of days to get verifications.

**Comment by ManikaDhiman** [comment]
Merging this PR now as changes are approved by the product team. I will create another PR to update "Windows Defender" in the GP names to "Microsoft Defender" in addition to several other updates. Thanks!

**Pull request merged**